form/icon-button.ftd

```diff
@@ -269,8 +269,23 @@ ftd.length.px border-radius: 8
 boolean $hover: false
 
 -- ftd.row:
-background.solid: $ds.colors.background.step-1
+background.solid if { button-wrapper.variant == solid }: $ds.colors.background.step-1
```

Comment:
I think this condition will be satisfied and there'd be no way to get into the rest of the conditions.